docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.6"
+
+services:
+  mysql:
+    image: mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: ${TFS_MYSQL_ROOT_PASSWORD:-r00tp@ssword}
```

Comment:
not the best default password

Reply:
Could you suggest what is the best default password?

Reply:
https://mariadb.com/kb/en/authentication-plugin-unix-socket/